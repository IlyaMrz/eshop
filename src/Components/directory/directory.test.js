import React from 'react';
import { shallow } from 'enzyme';
import { Directory } from './directory';

it('should render directory',()=>{
    expect(shallow(<Directory sections={[]} />)).toMatchSnapshot()
})