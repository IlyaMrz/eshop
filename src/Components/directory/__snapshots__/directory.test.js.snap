// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render directory 1`] = `<styled.div />`;
